prime-router/src/main/kotlin/cli/LivdTableDownload.kt

```diff
@@ -40,34 +42,62 @@ private const val sheetName = "LOINC Mapping"
 /**
  * LivdTableDownload is the command line interface for the livd-table-download command. It parses the command line
  * for option given as below.
+ *
+ * It looks for the LIVD-SAR-CoV-2-yyyy-MM-dd.xlsx file from $cdcLOINCTestCodeMappingPageUrl.  If the file is found,
+ * it downloads the file into the ./build directory.  If not found, it will prompt error accordingly.  Next, it build
+ * the output Lookup Table (<./build/LIVD-SARS-CoV-2.csv> file) with the table name.  Finally, it updates the
+ * LIVD-SARS-CoV-2 lookup tables in the database as the new version of the table.
+ * It updates new version of the lookup table in the given --env [local, test, staging, or prod] with the default
+ * to "local" environment.
+ *
+ * Note, this command will always create new version of the lookup table.
+ *
+ * Example:
+ * The command below will download the latest LIVD mapping catalogue and create a new lookup tables as needed.
+ * It took "LIVD mapping catalogue" from the CDC website.
+ *
+ *  ./prime livd-table-download
+ *
  */
-class LivdTableDownload : CliktCommand(
+class LivdTableDownload() : CliktCommand(
     name = "livd-table-download",
     help = """
-    livd-table downloads the latest LOINC test data, so it can be ingested automatically. 
-    
-    It looks for the LIVD-SAR-CoV-2-yyyy-MM-dd.xlsx file from $cdcLOINCTestCodeMappingPageUrl.
-	If the file is found, it downloads the file into the directory specified by the --output-dir <path> option.
-	If the option is not specified, it will download the file to ./build directory.
-        
-    Example: The following command will download the latest LIVD-SARS-CoV-2-yyyy-MM-dd.xlsx from the above URL.  
-      It will store the LIVD-SARS-CoV-2-xyz.csv file under the ./junk directory.
-      
-         ./prime livd-table-download --output-file ./junk/LIVD-SARS-CoV-2-xyz.csv
+    It downloads the latest LOINC test data, extract Lookup Table, and the database as a new version. 
     """
 ) {
     private val defaultOutputDir = "./build"
-    private val outputFile by option(
-        "--output-file",
-        metavar = "<path/filename>",
-        help = "Output pathname where to store the LIVD data as CSV."
-    ).required()
+    private val outPutFile = "./build/$livdSARSCov2File.csv"
```

Comment:
@oslynn I just noticed this small misspelling. `output` is all one word, `Put` shouldn't be cased like this.

Reply:
Done. Corrected misspelling and tested.